app/controllers/api/concerns/act_as_api_request.rb

```diff
@@ -10,7 +10,8 @@ module ActAsApiRequest
       end
 
       def check_json_request
-        return if request_content_type&.match?(/json/)
+        return unless request_with_body?
```

Comment:
Personally, I would find it easier to follow if it just had 1 early return.
Maybe something like:
```
return if request_without_body? || request_content_type.match?(/json/)
```

Reply:
Nice, looks good

Reply:
I think that what makes it hard to understand logicals expression is the use of both `if` and `unless` in the same method. I try not to use `unless` for that reason. I prefer the condition check to always be for the positive and negate the statement if necessary